devel/libprotozero/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                    1.0
+PortGroup           cmake                     1.1
+PortGroup           github                    1.0
```

Comment:
you don't really need to align the PG numbers; most ports would do this like this, which has in my opinion a slightly better readability but it's up to you.
```
PortSystem          1.0
PortGroup           cmake 1.1
PortGroup           github 1.0
```

Reply:
Agreed.  I'll conform to that too.